src/Helpers/Shared/userLogin.js

```diff
@@ -1,11 +1,11 @@
 import { AdminsApi, ApiClient as ServicePortalApiClient } from '@manageiq/service-portal-api';
 import { DefaultApi, ApiClient as TopologicalInventoryApiClient } from '@manageiq/topological_inventory';
```

Comment:
Minor, but the package names are inconsistent.   I know we haven't moved over the service-portal-api client to the ManageIQ repo, but I expected the client package names to be consistent.

Reply:
@Fryguy i agree